<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVDBInternalLoadsPersonEditWidget</class>
 <widget class="QWidget" name="SVDBInternalLoadsPersonEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>268</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelDisplayName">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QtExt::LanguageStringEditWidget1" name="lineEditName">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QtExt::ColorButton" name="pushButtonPersonColor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelCategory">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Level of Detail:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="comboBoxCategory">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelCategory_2">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Person Count Method:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QComboBox" name="comboBoxPersonMethod">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelPersonCount">
     <property name="text">
      <string>Max. Person Count:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditPersonCount">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="labelPersonCountUnit">
     <property name="text">
      <string>Person</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>195</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelDisplayName_2">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Occupancy Schedule Name:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEditOccupancyScheduleName">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QToolButton" name="toolButtonSelectOccupancy">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelDisplayName_3">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Activity Schedule Name:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEditActivityScheduleName">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QToolButton" name="toolButtonSelectActivity">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelConvectiveFactor">
     <property name="text">
      <string>Convective Factor:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditConvectiveFactor">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="labelConvectiveFactorUnit">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>195</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ValidatingLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QtExt::ColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">QtExt_ColorButton.h</header>
  </customwidget>
  <customwidget>
   <class>QtExt::LanguageStringEditWidget1</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_LanguageStringEditWidget1.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
